uint32 node_id
sensor_msgs/Image labels
rp_semantic/Cluster[] clusters


